examples/basic-click-counter/Index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>React click conter example</title>
+    <meta http-equiv='Content-type' content='text/html; charset=utf-8'>
+    <script src="../../build/react.js"></script>
+    <script type="text/javascript">
+    var Counter = React.createClass({
+      getInitialState: function () {
+        return { clickCount: 0 };
+      },
+      handleClick: function () {
+        this.setState(function(state) {
+          return {clickCount: state.clickCount + 1};
+        });
+      },
+      render: function () {
+        return React.DOM.h2({onClick: this.handleClick},
```

Comment:
Since jsx is the more idiomatic way of creating elements, I feel like we should probably use jsx in the examples, instead of manually creating the elements.  Thoughts @spicyj?